### DISCUSSION TITLE
Folder

### DISCUSSION THREAD

**Discussion by Vastolorde97** [shaked6540/YoutubePlaylistDownloader#117] [Ideas]
Is there anyway to make the app download to a specific folder or for it to make folder with the channels name when it downloads videos?

example 
LTT channel it makes an ltt channel folder and downloads the videos directly to it or
the LGR channel it makes an lgr channel folder and downloads to that folder

**Comment by shaked6540** [comment]
In the download options check the "Save every playlist in a different folder" checkbox. If you're downloading a channel it will use the channel name.
![image](https://user-images.githubusercontent.com/13851815/218880267-50301c1b-a109-4be3-b62b-5d2798ed6c0e.png)

**Reply by Vastolorde97** [reply]
can't seem to get that options page

**Reply by Vastolorde97** [reply]
![image](https://user-images.githubusercontent.com/59925365/218889241-5ba7c086-6b57-4996-8325-7b420c51079d.png)
only able to see this page

**Reply by shaked6540** [reply]
![image](https://user-images.githubusercontent.com/13851815/218890643-7256f5b5-375c-4a10-bae4-b30d12cb134f.png)
This button
